base/docker-compose.yaml

```diff
@@ -209,6 +212,7 @@ services:
     image: ghcr.io/uwcirg/shl-ltt-server:${SHL_SERVER_IMAGE_TAG:-latest}
     environment:
       PUBLIC_URL: https://shl-server.${BASE_DOMAIN}
+      VERSION: ${SHL_SERVER_IMAGE_TAG:-"latest"}
```

Comment:
The docker image tag won't provide much information on development servers, since they generally won't be tagged. You may want to use `git describe` output at build-time to populate this value.

Here's a PR the implements the CI, but you'll also need a way to consume that environment variable in the app code

uwcirg/asbi-screening-app#48

Reply:
Thanks! Added that function to the builds here:
shl-creator: https://github.com/uwcirg/shl-ltt/pull/27
shl-server: https://github.com/uwcirg/shl-ltt-server/pull/3
and removed the use of the image tag in this change.